packages/react-components/react-jsx-runtime/package.json

```diff
@@ -19,7 +19,7 @@
     "lint": "just-scripts lint",
     "test": "jest --passWithNoTests",
     "type-check": "tsc -b tsconfig.json",
-    "generate-api": "just-scripts generate-api",
+    "generate-api": "just-scripts copy && just-scripts generate-api",
```

Comment:
because non common flow of this package, `generate-api` wasn't providing proper build assets which failed on any dependant package `generate-api` target run.